setup.py

```diff
@@ -18,9 +18,9 @@
       ],
       py_modules=['tap_mysql'],
       install_requires=[
-          'pendulum==1.2.0',
+          'pendulum==1.5.1',
           'pipelinewise-singer-python==1.*',
-          'PyMySQL==0.7.11',
+          'PyMySQL>=0.7,<1.0',
```

Comment:
why this large range? it covers 5 years.

Reply:
We should not use a 5 years old package for sure, but going through the changelog there are some behavioural and backward imcompatible changes as well, even if it's still a minor version bump.

If tests are not covering them, that we might be impacted by the items:
* Formatting of float object is changed from "3.14" to "3.14e0".
* Use cp1252 codec for latin1 charset.
* Change default charset from latin1 to utf8mb4. 
* `BACKWARD INCOMPATIBLE` binary_prefix option is added and off by default because of compatibility with mysqlclient. When you need PyMySQL 0.7 behavior, you have to pass binary_prefix=True
* `BACKWARD INCOMPATIBLE MULTI_STATEMENTS` client flag is no longer set by default, while it was on PyMySQL 0.7. You need to pass client_flag=CLIENT.MULTI_STATEMENTS when you connect to explicitly enable multi-statement mode. 
```

Reply:
Urgh, I wrote an entire paragraph on why this change but forgot to click enter 🤦🏽 .

Anyway, yeah, those changelog look worrying, and I have reverted back.